bower.json

```diff
@@ -28,7 +48,6 @@
     "tests"
   ],
   "dependencies": {
-    "proj4": "^2.0.0",
-    "leaflet": "~0.7.2"
```

Comment:
Yes, it was removed after discussion in #127 - anyone who is using Proj4Leaflet now needs to get Leaflet first, independently.